README.md

```diff
@@ -2,7 +2,7 @@
 
 ## About
 
-Displays a block with links relative to a user's account.
+Enrich your stats, add a registration progress tab to analyze your visitors’ behavior.
```

Comment:
```suggestion
Enrich your stats, add a registration progress tab to analyze your visitor’s behavior.
```

Isn't it like this?

Reply:
@NeOMakinG I think we want it to be plural here so it is visitors' ☺️